app/components/player-card.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+	MilitaryTech,
+	TrendingUp,
+	TrendingUpOutlined
+} from "@mui/icons-material";
+import { Avatar, ButtonBase, Grid, Typography } from "@mui/material";
+import React from "react";
+import MaetIcon from "./maet-icon";
+
+export type PlayerCardProps = {
+	name: string;
+	position: number;
+	medalColor?: string;
+	score: number;
+	change: ScoreChange;
+};
+
+export type ScoreChange = {
```

Comment:
I would pull this out to the types dir and import it to this file if we are going to use this type often. If this is a one time type just add it to PlayerCardProps

`change: {color: string;
	magnitude: number;
}`